backend/lib/kubernetes-client/ApiErrors.js

```diff
@@ -55,9 +75,30 @@ function isExpiredError (err) {
   return isResourceExpired(err) || isGone(err)
 }
 
+function isTimeoutError (err) {
+  return err instanceof TimeoutError
+}
+
+function isTooLargeResourceVersionError (err) {
```

Comment:
link kubernetes client code

Reply:
Added this comment:
```js
/*
  In case of multiple apiserver instances the largest resourceVersion of watch caches
  will not be in sync anymore. This could lead to situations where we try to start a watch
  with a resourceVersion that is higher than the largest resourceVersion of particular watch cache.
  The apiserver will throw a TooLargeResourceVersionError in this case.
  https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go#L338-L344
*/
```